spec/expression.dd

```diff
@@ -2102,11 +2101,12 @@ void main()
     alias Tup = Tuple!(int, string);
     alias AA = long[char[]];
 
-    static if (is(Tup : TX!TL, alias TX, TL...))
+    static if (is(Tup : Template!Args, alias Template, Args...))
     {
-        writeln(is(TX!(int, long) == Tuple!(int, long)));  // true
-        writeln(typeid(TL[0]));  // int
-        writeln(typeid(TL[1]));  // immutable(char)[]
+        writeln(__traits(isSame, Template, Tuple)); // true
```

Comment:
@MetaLang For the repos for which dlang-bot is subscribed and have the `auto-merge-squash` label (I know of dlang-tour and dlang-community) I think it always keeps the message of the first commit and drops the messages of the subsequent commits.